app/js/redux-store.jsx

```diff
@@ -0,0 +1,22 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+import {createStore, applyMiddleware, combineReducers} from 'redux'
+import promiseMiddleware from 'redux-promise-middleware';
+import thunkMiddleware from 'redux-thunk';
+import * as reducers from './reducers'
+
+export default function () {
```

Comment:
I think we should use arrow functions (in general es6 syntax).

Reply:
Currently we have not yet started using redux and besides that those functions are autogenerated when you run the generator and can be changed later to arrow functions

Reply:
Great